packages/mdc-button/mdc-button.scss

```diff
@@ -87,110 +57,104 @@
     border: 0;
   }
 
-  &--dense {
-    height: 32px;
-    font-size: .8125rem; // 13sp
-    line-height: 32px;
+  &:not(.mdc-ripple-upgraded) {
+    -webkit-tap-highlight-color: rgba(98, 25, 130, .3);
```

Comment:
These are also referencing the same color that Lynn suggested storing in a variable.  Is there a built-in in sass that we could use to take the color stored in variable and apply alpha to it?